exonum/src/messages/signed.rs

```diff
@@ -198,6 +200,24 @@ impl<T> From<Verified<T>> for SignedMessage {
     }
 }
 
+impl<T> ProtobufConvert for Verified<T>
+where
+    T: TryFrom<SignedMessage>,
+{
+    type ProtoStruct = schema::consensus::Verified;
```

Comment:
Micro-nit: `schema` is a bit ambiguous because of the `merkledb`. I think you can just use re-export in `proto` module and use `proto::Verified` here.